src/main/java/com/example/demo/response/dto/UserDto.java

```diff
@@ -0,0 +1,99 @@
+package com.example.demo.response.dto;
+
+import java.io.Serializable;
+
+import org.springframework.stereotype.Component;
+
+@SuppressWarnings("serial")
+@Component
+public class UserDto implements Serializable {
+	private String name;
+	private String mail;
+	private String designation;
+	private String project;
+	private String oracleid;
+	private String seat;
+	private String img;
+	private String imgurl;
+	private String extension;
+	private String du;
+	
+	public UserDto() {
+		// TODO Auto-generated constructor stub
+	}
+	public UserDto(String name, String mail, String designation, String project, String oracleid, String seat,
+			String img, String imgurl, String extension, String du) {
+		super();
+		this.name = name;
```

Comment:
remove super()

Reply:
Removed, But what's the point of removing it. Is it gonna make any difference?

Reply:
Their is no need to declare it explictly